firefox-ios/Client/AccessoryViewProvider.swift

```diff
@@ -7,61 +7,56 @@ import Common
 import Shared
 
 enum AccessoryType {
-    case standard, creditCard
+    case standard, creditCard, address
 }
 
 class AccessoryViewProvider: UIView, Themeable {
+    // MARK: - Constants
     private struct UX {
         static let toolbarHeight: CGFloat = 50
-        static let cornerRadius: CGFloat = 4
-        static let cardImageViewSize: CGFloat = 24
         static let fixedSpacerWidth: CGFloat = 10
         static let fixedSpacerHeight: CGFloat = 30
         static let fixedLeadingSpacerWidth: CGFloat = 2
         static let fixedTrailingSpacerWidth: CGFloat = 3
-        static let cardButtonStackViewSpacing: CGFloat = 2
     }
 
+    // MARK: - Properties
     var themeManager: ThemeManager
     var themeObserver: NSObjectProtocol?
     var notificationCenter: NotificationProtocol
-    private var showCreditCard = false
+    private var currentAccessoryView: AutofillAccessoryViewButtonItem?
 
-    // stubs - these closures will be given as selectors in a future task
+    // Stub closures - these closures will be given as selectors in a future task
     var previousClosure: (() -> Void)?
     var nextClosure: (() -> Void)?
     var doneClosure: (() -> Void)?
     var savedCardsClosure: (() -> Void)?
+    var savedAddressesClosure: (() -> Void)?
 
-    private var toolbar: UIToolbar = .build { toolbar in
-        toolbar.sizeToFit()
+    // MARK: - UI Elements
+    private let toolbar: UIToolbar = .build {
+        $0.sizeToFit()
     }
 
     private lazy var previousButton: UIBarButtonItem = {
```

Comment:
Should we use `.build ` for UIBarButtonItem or is it not available?

Reply:
Good point @jnrahme did the build not work? 

Usually build is for standard inits

Reply:
Going to r+ it regardless but try it